chromium_src/chrome/browser/ui/tabs/tab_strip_model.h

```diff
@@ -8,8 +8,12 @@
 
 #define SelectRelativeTab virtual SelectRelativeTab
 #define TAB_STRIP_MODEL_H_ friend class BraveTabStripModel;
+#define IsReadLaterSupportedForAny     \
+  IsReadLaterSupportedForAny_Unused(); \
```

Comment:
a simpler replacement like `virtual IsReadLaterSupportedForAny` doesn't work here?

Reply:
Fixed in 5ecd21a2fdf2b38cd2ec1eb377fa7c0b2cbea7e1